### PR TITLE
Winter Landing

### PR DESCRIPTION
**Pull request by dasilvacontin**
Still missing recolor.

Also, snowflakes are getting blurred. We need to fix that. And maybe improvements to the code, because the laptop gets hot.

## Tasks Left:
- [x] snowflakes should not be blurred (not necessary for PR merge, can do later)
- [x] copy (text) updates (update 2016->2017, fall->winter, etc)
- [x] update social image's colors (update social images eg og:share favicon etc with new palette)
- [ ] add YouTube video preview

### CODE REVIEW

File: src/scripts/leaves.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Leaves {
 
   constructor(public constantOpacity: boolean) {
     this.loadLeaves();
-    this.renderer = PIXI.autoDetectRenderer(800, 600, {backgroundColor : 0xfdf3e1, antialias: false});
+    this.renderer = PIXI.autoDetectRenderer(800, 600, {backgroundColor : 0x0000ff, antialias: false});
```
Comment (casassg) [inline src/scripts/leaves.ts]: This background color? New color schema -> HackUPC Winter 2016 reverse?

File: src/scripts/leaves.ts [hunk 2]
```diff
@@ -114,22 +114,20 @@ class Leaves {
...
+    PIXI.loader.load(() => this.createLeaves());
   }
 
   private createLeaves() {
     for(var i = 0; i < this.maximumLeaves; i++) {
-      var leafIdx = Math.round(Math.random()*50) % 4;
+      var flakeId = Math.floor(Math.random() * 13)
       var posX = Math.random() * this.renderer.view.width;
       var posY = Math.random() * this.renderer.view.height;
       var radius = this.minSize + Math.random() * this.maxSize + 1;
```
Comment (casassg) [inline src/scripts/leaves.ts:120]: I would reduce maxSize, snowflakes seem too big. And add more snoflakes to compensate

**Comment by casassg** [comment]
To avoid the blur I would decrease the size of snowflakes and increase the total number of them being shown at the same time.

File: src/scripts/leaves.ts (hunk 1 above)
Comment (dasilvacontin) [reply src/scripts/leaves.ts]: No, just testing. PR doesn't mean it's final. Only snowflake sprites are relevant for now.

File: src/scripts/leaves.ts (hunk 2 above)
Comment (dasilvacontin) [reply src/scripts/leaves.ts:120]: No idea, I haven't looked into that yet. I did notice they seemed to fall too fast for my taste.

**Comment by dasilvacontin** [comment]
> To avoid the blur I would decrease the size of snowflakes and increase the total number of them being shown at the same time.

I think the blur is due to canvas / PIXI settings. Avoidable.

**Comment by dasilvacontin** [comment]
Obvs el azul del principio degrada al hacer scroll, pero la tool hace screenshots a trozos.

screencapture-localhost-8080-1482354494569

**Comment by casassg** [comment]
What about header in white? (include here top buttons and subtitles)

**Comment by dasilvacontin** [comment]
screencapture-localhost-8080-1482426062681

**Comment by dasilvacontin** [comment]
Still needs copy / content changes.

**Comment by dasilvacontin** [comment]
From Slack:

> @casassg: @dasilvacontin I can PR content updates
We will need logos and color palette though
Also, did you fix the blurriness in the snowflakes?

Not sure what logos you are talking about.
Colors are in variables at the top of the SASS file, but I can extract them here for convenience.
Blurriness is still an issue.

**Comment by dasilvacontin** [comment]
Letting you know I'm busy on the 29th and the 30th.

**Comment by casassg** [comment]
By logos I mean, the HackUPC images. They need to be updated with the new theme:

- https://github.com/hackupc/hackupc/blob/master/src/images/hackupc-ogimage%402x.png
- https://github.com/hackupc/hackupc/blob/master/dist/favicon.ico

Also, adding in this branch content updates

**Comment by dasilvacontin** [comment]
@elena20ruiz Merci! Btw, no hace falta crear una nueva pull – con hacer push de commits a la branch de esta PR es suficiente. :)

**Comment by dasilvacontin** [comment]
@elena20ruiz I'm currently available to work on it and finish it. Do you mind pushing whatever you have so that I can continue from there, or what do you suggest?

**Comment by dasilvacontin** [comment]
@GonMolon, subscribe to this thread if you want to get any status updates about it.

**Comment by dasilvacontin** [comment]
@elena20ruiz Cool! `hackupc-ogimage@2x.png` needs a background color though, it shouldn't be transparent. Otherwise, depending on the app, it might get a white/black background.

This image is used as the preview image for when someone shares the link e.g. on fb, telegram.

**Comment by dasilvacontin** [comment]
@GonMolon, we could use some input / review from you. Should we list ANY sponsor? Should we list ANY partner?

**Comment by dasilvacontin** [comment]
I'll probably embed a youtube video from last edition, as a preview.

**Comment by elena20ruiz** [comment]
@dasilvacontin  Justo lo había pensado. Entonces hago 2 versiones entiendo. 

Otra cosa que no comenté cuando tratabais de escoger la paleta, era si podías probar con los siguientes colores. 
Como fondo #068D9D
En cambio tener dos tonalidades de 'snowflakes', combinar blanco con #50BAD5

Pero si es así no se como quedaria el rosa en combinación.

**Comment by dasilvacontin** [comment]
@elena20ruiz ahora los pruebo! :)

**Comment by dasilvacontin** [comment]
@elena20ruiz Hmm, con #068D9D tendría que ser una paleta nueva – no se lleva bien con los colores de la actual.

Note: no estoy en contra de cambiar la paleta. Pero dado que esta paleta / página será provisional, y dado que ya es dia 2, creo que deberiamos 'tirar adelante' con la actual. Y luego cambiarla más adelante si fuera necesario.

Ahora pruebo la otra tonalidad de snowflake.

**Comment by dasilvacontin** [comment]
I added https://github.com/hackupc/hackupc/pull/63/commits/b972336fffdca4e7eee86106d36501a53ea061bf with your proposed blue-ish snowflakes so that we can play with it. We can revert it (and whatever experiments we make) if we end up not liking it or it doesn't improve the design.

Current result as-is isn't an improvement imo. Randomly changing the `alpha` of the flakes could be tried too.

screen shot 2017-01-01 at 11 51 11 pm

**Comment by dasilvacontin** [comment]
(Also, just a note, that all the snowflakes I added are just for testing, we can get rid of any we don't like)

**Comment by dasilvacontin** [comment]
Okay @GonMolon, I don't see any mention of fall/2016 left on the text. I need input from you on the sponsors/partners topic. What to leave, if any, or to get rid of the sections altogether for now.

I need links for the 'Apply' and the 'Sponsor' button too.

I assume sponsor link will remain mailto:sponsor@hackupc.com, but just in case you want to add something different.

**Comment by dasilvacontin** [comment]
(random-note: I find this FAQ structure very hard to read. There's no clear "direction" to follow when reading)

**Comment by dasilvacontin** [comment]
Okay. Now just waiting for @GonMolon's input, or someone else in the position of answering those questions. I'll add youtube video after those get answered, since it may affect the layout.

@elena20ruiz, is it me or `src/images/hackupc-ogimage@2x.png` doesn't have the same bg color as the website? (#676ff1) Was that an intended choice? Or does the page's background look different than #676ff1 for you, maybe? Just to be on the same page. Also, if there's any 'source' file that generated that image, please add it to `/design/`, for future needs :)

**Comment by gonsp** [comment]
@dasilvacontin Sorry tío, no tengo mucho internet y no he respondido antes. Tampoco puedo probar la web porque no tengo npm instalado y no se me descarga. Respecto al diseño, por lo que he visto me parece guay y me fío de vuestro criterio. Como comentario sobre el diseño, al probar la web hace unos días al hacer scrolling, los snowflakes se confunden con el fondo. Sería fácil hacer que el color de los snowflakes vayan cambiando a un azul a medida que el fondo pasa a blanco (que se inviertan los colores)?
Con el resto de cosas:
- Sponsors: Que el título sea past sponsors. Cuando tengamos confirmados los nuevos ya lo cambiaremos.
- Link mail sponsors: No se cambia
- Link apply: https://hackersatupc.typeform.com/to/JCVBqv
- El texto lo actualizaría y añadiría el comentario de la pasada edición. Mañana por la noche lo tendré preparado para que lo cambiéis.

**Comment by dasilvacontin** [comment]
> Como comentario sobre el diseño, al probar la web hace unos días al hacer scrolling, los snowflakes se confunden con el fondo.

@GonMolon Esa era la intención, que pasaran desapercibidos, para no distraer. Se puede probar algo diferente después del ship inicial. No sería muy difícil, pero es tiempo.

> Que el título sea past sponsors

top 👌 

> El texto lo actualizaría y añadiría el comentario de la pasada edición. Mañana por la noche lo tendré preparado para que lo cambiéis.

Alrighta. 🙆‍♂️

**Comment by gonsp** [comment]
@dasilvacontin A qué te referías con lo de revisar el FAQ? Era para actualizar el texto o tambien el cómo se muestra (layout)?

**Comment by gonsp** [comment]
Habéis mencionado algo de meter un video de youtube? Sería guay el trailer o el de la última edición incrustado en la web, no solo con el botón de vídeos!

**Comment by dasilvacontin** [comment]
> Habéis mencionado algo de meter un video de youtube? Sería guay el trailer o el de la última edición incrustado en la web, no solo con el botón de vídeos!

Yep https://github.com/hackupc/hackupc/pull/63#issuecomment-269922306

**Comment by dasilvacontin** [comment]
> @dasilvacontin A qué te referías con lo de revisar el FAQ? Era para actualizar el texto o tambien el cómo se muestra (layout)?

El texto. El layout, aunque no me convence, se quedará así por ahora >.<

**Comment by gonsp** [comment]
@dasilvacontin Aquí está el nuevo texto: https://docs.google.com/document/d/1n5ekSKG8Pw2ClbdVREG0Mc3xHN5pxddhAkjYRsbgk-M/edit

Muy buen trabajo a todos!

**Comment by dasilvacontin** [comment]
No puedes hacer commit a la branch del PR?

**Comment by dasilvacontin** [comment]
@AsLogd ty much for snowflake fix. I tried something like that, but before PIXI/scene creation. That was the problem, I think.

**Comment by AsLogd** [comment]
@dasilvacontin np. I got it right by accident then, I don't know pixi. But it was clear it had something to do with interpolation, yes

**Comment by dasilvacontin** [comment]
@casassg why remove some sponsors if the section is called 'past sponsors'?

**Comment by casassg** [comment]
I talked with Gonzalo. Removing some if the smaller sponsor seems like a
good option to keep the page smaller (and to improve loading time).

Regarding the video, I'm pinging the people at RebelUncut to see if we can
get a teaser in time. Otherwise Highlights video seems like a good option.

**Comment by dasilvacontin** [comment]
btw, when merging, don't squash, keep history, ty

**Comment by casassg** [comment]
@elena20ruiz could you upload here the last design too pls

**Comment by casassg** [comment]
@dasilvacontin should we merge master here? I'm trying to add .travis.yml but running configuration differ from master to here

**Comment by dasilvacontin** [comment]
@casassg I prefer rebasing, unless it modifies the dates of the commits. I don't care too much, but it would be nicer if they remain the same. And I'm not sure if that's the case atm.

**Comment by casassg** [comment]
Oops sorry @dasilvacontin didn't see the comment up there. Will do rebase next time

**Comment by casassg** [comment]
Can someone review last changes?

**Comment by dasilvacontin** [comment]
@casassg why bring back the sponsors?

**Comment by casassg** [comment]
@dasilvacontin ask @quim-motger @mllobet and fertermar

> tl;dr Not good to have only some: all or none

File: src/scripts/snowflakes.webgl.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
+/// <reference path="typings/pixi.js/pixi.js.d.ts"/>
+var MAX_OPACITY = .9;
```
Comment (dasilvacontin) [inline src/scripts/snowflakes.webgl.js]: Isn't this file autogenerated?

Comment (dasilvacontin) [reply src/scripts/snowflakes.webgl.js]: Update the `.gitignore` please.

**Review by dasilvacontin — COMMENTED** [review]
All good except for that autogenerated file.

**Comment by dasilvacontin** [comment]
Btw, since I'm an author of this PR, I can't approve it, per GitHub rules.

**Comment by dasilvacontin** [comment]
> @casassg : tl;dr Not good to have only some: all or none

Sounds logical.

**Comment by dasilvacontin** [comment]
Still missing that YT video.

**Comment by casassg** [comment]
We will add the video once it's done. I expect that it will be in a few weeks. This way we can even reach more people

**Review by casassg — APPROVED** [review]
ready to :sheep:
